CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Non-protocol Changes
 
+* Enforce rate limits to received network messages [#11617](https://github.com/near/nearcore/issues/11617). Rate limits are configured by default, but they can be overridden through the experimental configuration option `received_messages_rate_limits`.
```

Comment:
here I call it experimental because network overrides are inside the `experimental`  object in the config: [example](https://github.com/near/nearcore/issues/11617#issuecomment-2188408709)

I can remove this wording if we don't consider network overrides experimental configuration